src/pact.ts

```diff
@@ -146,10 +150,19 @@ export class Pact {
     this.finalized = true;
 
     return this.mockService.writePact()
-      .then(() => this.server.delete())
-      .catch((err: Error) => {
-        return Promise.all([this.server.delete(), Promise.reject(err)]);
-      });
+      .then(() => logger.info("Pact File Written"), (e) => {
+        console.log("Pact File Write Failure");
```

Comment:
Any reason for the `console.log` independent of the log?

Reply:
Are they just debug statements during dev that needed to go?

Reply:
ah yeah, sorry, missed that one.